CGPs/cgp-0060.md

```diff
@@ -19,6 +19,18 @@ As an interim solution to offer cUSD-USDC liquidity short term, before GigaMento
 </p>
 *depending on how the pool is utilized, governance might add more liquidity in the future via additional governance proposals until GigaMento comes into effect, see [Rebalancing](#rebalancing-16).
 
+## Market Dynamics and Reserve Balance
+
+## Arbitrage and DEX Competition
+The setup of the StableSwap pool is competitive. The transactions fee is on a similar level as for stable pairs on other DEXs and additionally it uses a constant sum market making function. As a consequence, especially larger transactions will be cheaper with the constant sum StableSwap pool.
+
+Arbitrageurs will take advantage of the arbitrage opportunity. In the current market situation (cUSD/USDC price at ~0.995 ), arbitrageurs will buy cUSD below 1 USD in the market and sell it to the StableSwap pool for a cUSD/USDC price of 1. Ignoring fees and slippage, the maximal profit they can earn if cUSD/USD remains at 1 is
+
+$$ 5\,000\,000 (1-0.995) = 25\,000\mathrm{\ USDCet} \ .$$
+
+## Reserve Balance
+The reserve will always be able to unwind
+irrespective of whether arbitrageurs take arbitrage opportunities or not. Under the assumption that the USDC/USD and cUSD/USD prices stay a 1, the net reserve balance (reserve collateral minus debt) will not change.
```

Comment:
```suggestion
As long as USDC/USD and cUSD/USD stay a 1, the reserve will always be able to unwind without a net loss - irrespective of whether arbitrageurs take arbitrage opportunities or not. This is because exchanges happen 1:1 and therefore changes in reserve debt and changes in reserve collateral cancel each other out.
```